lib/Twig/TemplateWrapper.php

```diff
@@ -102,26 +102,7 @@ public function renderBlock($name, $context = array())
      */
     public function displayBlock($name, $context = array())
     {
-        $context = $this->env->mergeGlobals($context);
-        $level = ob_get_level();
-        ob_start();
-        try {
-            $this->template->displayBlock($name, $context);
-        } catch (Exception $e) {
-            while (ob_get_level() > $level) {
-                ob_end_clean();
-            }
-
-            throw $e;
-        } catch (Throwable $e) {
-            while (ob_get_level() > $level) {
-                ob_end_clean();
-            }
-
-            throw $e;
-        }
-
-        return ob_get_clean();
+        $this->template->displayBlock($name, $this->env->mergeGlobals($context));
```

Comment:
you want my fix of `display()` in there as well, than I can revert that change here as well and have no more changes on the file in this PR